modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+Support for expanding CSI volumes is enabled by default, but it also requires a specific CSI driver to support volume expansion.
+
+:FeatureName: Expanding CSI volumes
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+[IMPORTANT]
```

Comment:
I'd really try to break as much information as possible out of the admonition. https://stylepedia.net/style/#admonitions

Reply:
I was reusing an existing admonition (from another assembly) that explains the same functionality. But I agree, so I removed this admonition formatting altogether and made the info part of the non-formatted body.